VideoLocker/res/layout/player_controller.xml

```diff
@@ -1,224 +1,204 @@
             android:id="@+id/share_btn"
-            android:layout_width="20dp"
-            android:layout_height="20dp"
-            android:layout_centerVertical="true"
-            android:layout_toLeftOf="@id/lms_btn"
-            android:layout_toStartOf="@id/lms_btn"
-            android:layout_marginRight="10dp"
-            android:layout_marginEnd="10dp"
-            android:background="@drawable/ic_facebook_discuss_selector"
-            android:contentDescription="@string/app_name"
-            tools:targetApi="17"
-            />
-                
-    </RelativeLayout>
+            style="@style/video_player_top_control"
+            android:layout_width="@dimen/video_player_top_control_width"
+            android:layout_height="@dimen/video_player_top_control_height"
+            android:contentDescription="@string/video_player_share"
+            app:iconColor="@color/edx_grayscale_neutral_white_t"
+            app:iconName="fa_facebook_square"
+            tools:targetApi="17" />
+    </LinearLayout>
 
     <LinearLayout
         android:id="@+id/panel_bottom"
+        style="@style/video_player_panel"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_alignParentBottom="true"
+        android:layout_gravity="bottom"
         android:layoutDirection="ltr"
-        android:background="#CC000000"
-        android:orientation="vertical"
-        android:padding="7dp" >
+        android:orientation="horizontal"
+        android:padding="7dp"
+        tools:targetApi="17">
+
+        <org.edx.mobile.third_party.iconify.IconButton
+            android:id="@+id/rew"
+            style="@style/video_player_bottom_control"
+            android:layout_width="@dimen/video_player_bottom_control_width"
+            android:layout_height="@dimen/video_player_bottom_control_height"
+            android:contentDescription="@string/video_player_rewind"
+            app:iconColor="@color/edx_grayscale_neutral_white_t"
+            app:iconName="fa_history"
+            tools:targetApi="17" />
+
+        <SeekBar
+            android:id="@+id/mediacontroller_progress"
+            android:layout_width="0dp"
+            android:layout_height="24dp"
+            android:layout_marginEnd="6dp"
+            android:layout_marginRight="6dp"
+            android:layout_weight="1"
+            android:background="@drawable/ic_seek_background"
+            android:max="100"
+            android:maxHeight="24dp"
+            android:minHeight="24dp"
+            android:paddingEnd="0dp"
+            android:paddingLeft="0dp"
+            android:paddingRight="0dp"
+            android:paddingStart="0dp"
+            android:progress="30"
+            android:progressDrawable="@drawable/player_seekbar_progress"
+            android:secondaryProgress="70"
+            android:thumb="@drawable/player_seekbar_thumb"
+            android:thumbOffset="0dp"

```

Comment:
Should these attributes be moved to styles as well? Currently there's only one instance of it, but since everything else has been exported to styles, this might be as well. At least the dimensions should be exported.